pages/having-a-baby.js

```diff
@@ -100,6 +105,8 @@ export default function lifejourney() {
             />
           </div>
 
+          {/* Journey Stages Section */}
+
           <div className="container flex flex-col w-full bg-gray-200 md:flex-row align-items-center mt-4">
             <div className="p-2">
               <h2 className="text-h3-tall">
```

Comment:
The Figma designs have the guided journey component inside the gray container as well. Maybe do something like this?

```suggestion
          <div className="p-4 w-full bg-gray-200 md:flex-row align-items-center mt-4">
            <div className="p-2">
              <h2 className="text-h3-tall">
                Discover the stages of the new child life Journey
              </h2>

              <p>Learn more about the keytask during each stage in journey</p>
            </div>

            <div className="container mt-4">
              <Stages journeys={journeysData} />
            </div>
          </div>
```